pkg/k8s/watchers/watcher.go

```diff
@@ -141,6 +141,8 @@ type policyRepository interface {
 
 type svcManager interface {
 	DeleteService(frontend loadbalancer.L3n4Addr) (bool, error)
+	GetDeepCopyServices() []*loadbalancer.SVC
```

Comment:
We can remove this from the interface now, right?

Reply:
removed!